kalite/topic_tools/content_models.py

```diff
@@ -416,19 +425,14 @@ def recurse_availability_up_tree(node, available):
                     recurse_availability_up_tree(parent, available)
 
 
-            for path, update in updates_dict.iteritems():
+            for path, update in content_models:
                 if update:
                     # We have duplicates in the topic tree, make sure the stamping happens to all of them.
-                    items = Item.select().where(Item.path == path)
+                    items = Item.select().where((Item.path == path) & (Item.kind != "Topic"))
+
                     for item in items:
-                        if item.kind != "Topic":
-                            item_data = unparse_model_data(model_to_dict(item, recurse=False))
```

Comment:
Deleting these lines makes this whole thing a noop. You aren't actually updating any of the content nodes.